cid_test.go

```diff
@@ -448,7 +531,7 @@ func TestParse(t *testing.T) {
 			return err
 		}
 		if cid.Version() != 0 {
-			return fmt.Errorf("expected version 0, got %s", string(cid.Version()))
+			return fmt.Errorf("expected version 0, got %s", fmt.Sprintf("%d", cid.Version()))
```

Comment:
fmt.Errorf followed by fmt.Sprintf is a bit redundant :) you can just use %d in the Errorf